edvart/report_sections/univariate_analysis.py

```diff
@@ -63,6 +63,7 @@ def required_imports(self) -> List[str]:
             ]
         # Verbosity.HIGH
         return [
+            "import numpy as np",
```

Comment:
shouldn't this be a different PR with `fix` type?

Reply:
Done in #175